Cheers.java

```diff
@@ -1,6 +1,23 @@
 // Prints a crowd cheering output.
+
 public class Cheers {
         public static void main(String[] args) {
-	    // Replace this comment with your code
+            String inputOfCheer = args[0];
+            int timesToCheer = Integer.parseInt(args[1]);
+            String lettersToMatch = "AaEeFfHhIiLlMmNnOoRrSsXx";
+	    for (int i = 0; i < inputOfCheer.length(); i++) {
+                char inputInUpperCase = inputOfCheer.charAt(i);
+                inputInUpperCase = Character.toUpperCase(inputInUpperCase);
+                // Character.toUpperCase(inputInUpperCase);
+                if (lettersToMatch.indexOf(inputOfCheer.charAt(i))>=0) {
+                    System.out.println("Give me an " + inputInUpperCase + ": " + inputInUpperCase + "!");
```

Comment:
see styling guidelines

Reply:
Is that we you meant?
`if (lettersToMatch.indexOf(inputOfCheer.charAt(i)) >= 0) {`

Reply:
1. Yes
2. Regarding moving to next line.. how to work with it..